staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/controller.go

```diff
@@ -353,15 +355,30 @@ func (c *celAdmissionController) Validate(
 			}
 
 			for _, decision := range decisions {
+				paramStr := ""
+				if paramKind != nil {
+					gv, err := schema.ParseGroupVersion(paramKind.APIVersion)
+					if err == nil {
+						// only provide paramStr if we can successfully parse the groupversion
+						gvk := gv.WithKind(paramKind.Kind)
+						paramStr = gvk.String()
+					}
+				}
 				switch decision.kind {
 				case admit:
-					// TODO: add metrics for ignored error here
+					if decision.message != "" {
+						// the decision message is set in the admit case if and only if
+						// an ignored error occurred, so use that to filter for ignored
+						// errors (as opposed to validations that occurred with no errors)
+						celmetrics.Metrics.ObserveAdmission(ctx, decision.elapsed, definition.Name, binding.Name, decision.expression, paramStr, "active")
+					}
 				case deny:
 					deniedDecisions = append(deniedDecisions, policyDecisionWithMetadata{
 						definition:     definition,
 						binding:        binding,
 						policyDecision: decision,
 					})
+					celmetrics.Metrics.ObserveRejection(ctx, decision.elapsed, definition.Name, binding.Name, decision.expression, paramStr, "active")
```

Comment:
Same question. Is paramStr needed?

Reply:
(resolving due to https://github.com/kubernetes/kubernetes/pull/113475#discussion_r1016954715)